webrtc.html

```diff
@@ -1312,6 +1312,13 @@ <h4>
                 </p>
               </li>
               <li>
+                <p>
+                  Let <var>connection</var> have a
+                  <dfn data-dfn-for="RTCPeerConnection">[[\DataChannels]]</dfn> internal slot, initialized to
+                  an empty [=ordered set=].
```

Comment:
Why "ordered set" in particular?

Reply:
This appears to be recommended, and there's no infra for unordered sets. From https://infra.spec.whatwg.org/#sets:

<img width="957" alt="image" src="https://github.com/w3c/webrtc-pc/assets/3136226/e70e7086-8cf1-4443-8994-369ff52569a4">